ui/app/AppLayouts/Profile/views/WalletView.qml

```diff
@@ -218,12 +248,15 @@ SettingsContentBase {
         }
 
         ManageTokensView {
-            Layout.fillWidth: true
-            Layout.leftMargin: Style.current.padding
-            Layout.rightMargin: Style.current.padding
-
+            id: manageTokensView
             sourcesOfTokensModel: tokensStore.sourcesOfTokensModel
             tokensListModel: tokensStore.extendedFlatTokensModel
+            baseTokensModel: RootStore.assets // TODO concat proxy model to include community assets (#12369)
+            baseCollectiblesModel: {
+                RootStore.setFillterAllAddresses() // FIXME no other way to get _all_ collectibles?
```

Comment:
Could you create (if not yet created) a separate task to review how to do this integration?

Reply:
Maybe @dlipicar or @Cuteivist have an idea? Basically we want to see all the collectibles from all the addresses (unless they are hidden) in a flat model

Reply:
Sure, I think we need to discuss that and try to have a similar approach than with assets. We will discuss it later with @dlipicar !